deployment/setup_solana_for_tests.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
Let's make this portable

```suggestion
#!/usr/bin/env bash
```